File: scripts/template.php
<?php
$script_file_folder = __DIR__ . '\\';
echo $script_file_folder;


require_once($script_file_folder . 'modules\\php\\Template.php');

$template = new Template;
if ($template->main(null)) {
	print("\nOK\n");
} else {
	print("\nError\n");
}

File: scripts/template.py
# -*- coding: utf-8 -*

import sys
from os.path import dirname
from os.path import abspath

script_file_folder = abspath(dirname(sys.argv[0])) + '\\'
MY_TOOLS_MODULES_FOLDER = script_file_folder + 'modules\\python'
sys.path.insert(0, MY_TOOLS_MODULES_FOLDER)

from modules.python.Template import *

template = Template()
if template.main(sys.argv):
	print("OK\n")
else:
	print("Error\n")